defaults/main.yml

```diff
@@ -1,5 +1,11 @@
 ---
+postgres_version: 9.6
 postgres_locale: en_US.UTF-8
+postgres_extra_packages:
+  - python3-psycopg2
```

Comment:
Unless the python packages are required for our role to work, I don’t think we should suggest they be included.

Reply:
Ok, I moved these to the example-vars in the test playbook.